triggers/close-gmail-tabs/screen-share-started

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ -n "$TUPLE_TRIGGER_IS_SELF" ]; then
+  osascript -l JavaScript <<'END'
+    var chrome = Application('Google Chrome');
+    if (chrome.running()) {
+        var windows = chrome.windows();
+        for (var i = 0; i < windows.length; i++) {
+            var tabs = windows[i].tabs();
+            for (var j = tabs.length - 1; j >= 0; j--) {
+                var tab = tabs[j];
+                if (tab.url().includes("mail.google.com")) {
```

Comment:
This has the potential to cause some false positives if for whatever reason another URL happened to contain `mail.google.com`. Could we change this to something a touch more specific, e.g.

```suggestion
                if (tab.url().startsWith("https://mail.google.com/")) {
```

Once that's updated this should be good to go!